src/modules/commander/px4_custom_mode.h

```diff
@@ -61,7 +61,7 @@ enum PX4_CUSTOM_SUB_MODE_AUTO {
 	PX4_CUSTOM_SUB_MODE_AUTO_MISSION,
 	PX4_CUSTOM_SUB_MODE_AUTO_RTL,
 	PX4_CUSTOM_SUB_MODE_AUTO_LAND,
-	PX4_CUSTOM_SUB_MODE_AUTO_RTGS,
```

Comment:
This is a problem - the value still needs to be there so that the other modes do not move by one. I would just rename it to reserved, but leave it in.